crates/biome_cli/src/execute/migrate/eslint_eslint.rs

```diff
@@ -184,14 +184,14 @@ impl GlobalConf {
 }
 impl Deserializable for GlobalConf {
     fn deserialize(
+        ctx: &mut impl DeserializableContext,
         value: &impl biome_deserialize::DeserializableValue,
         name: &str,
-        diagnostics: &mut Vec<biome_deserialize::DeserializationDiagnostic>,
     ) -> Option<Self> {
         if value.visitable_type()? == DeserializableType::Str {
-            Deserializable::deserialize(value, name, diagnostics).map(Self::Qualifier)
+            Deserializable::deserialize(ctx, value, name).map(Self::Qualifier)
```

Comment:
What's the reason you changed the order of parameters? We shouldn't do it unless there is a strong reason for it.

Reply:
Do you think we should avoid flipping parameters?

Reply:
No, I think that's fine. You're right, usually the `ctx` is provided as the first parameter